### ISSUE TITLE
Capture any `window` or `document` properties that are useful.

### ISSUE THREAD

**Issue by joshed-io** [keen/common-web#1]
Thanks for the suggestion Cole!

**Comment by josephwegner** [comment]
Got any context here? What sort of properties are we thinking?

Stuff I can think of that might be interesting:
- screen size
- scroll position
- title

**Comment by dzello** [comment]
Yeah, I think those are good. Thinking they could be helpful for drawing heatmaps or doing analytics on display formats, etc.

**Comment by josephwegner** [comment]
Query params would also be nice - specifically for utm stuff.

**Comment by dzello** [comment]
For sure. iirc the keen URL parser should grab them. I updated the example JSON in the README to reflect that.
